tests/integration/models/reddit/test_emoji.py

```diff
@@ -1,8 +1,9 @@
-from praw.exceptions import ClientException
-from praw.models import Emoji
 import mock
 import pytest
 
+from praw.exceptions import ClientException
```

Comment:
I'm not quite sure why there are 4 import sections (global, third-party, package under test?, local). I usually treat the package under test as `third-party` since I haven't seen any documentation to suggest doing it otherwise.

Is this the default behavior of isort? If so, I suppose we can keep it that way.

Reply:
From their documentation:

isort parses specified files for global level import lines (imports outside of try / except blocks, functions, etc..) and puts them all at the top of the file grouped together by the type of import:

- Future
- Python Standard Library
- Third Party
- Current Python Project
- Explicitly Local (. before import, as in: from . import x)
- Custom Separate Sections (Defined by forced_separate list in configuration file)
- Custom Sections (Defined by sections list in configuration file)

Inside of each section the imports are sorted alphabetically. isort automatically removes duplicate python imports, and wraps long from imports to the specified line length (defaults to 79).